core/src/main/scala/cromwell/core/path/PathBuilderFactory.scala

```diff
@@ -15,11 +15,7 @@ object PathBuilderFactory {
     // The DefaultPathBuilderFactory always needs to be last.
     // The reason is path builders are tried in order, and the default one is very generous in terms of paths it "thinks" it supports
     // For instance, it will return a Path for a gcs url even though it doesn't really support it
-    val sortedFactories = factories.sortWith({
-      case (_, DefaultPathBuilderFactory) => true
-      case (DefaultPathBuilderFactory, _) => false
-      case (a, b) => factories.indexOf(a) < factories.indexOf(b)
-    })
+    val sortedFactories = factories.sortBy(_.priority)
```

Comment:
This is really much easier to think about - Nice! Does this still guarantee that the default is last?

Reply:
I gave the default a very low priority here: https://github.com/broadinstitute/cromwell/pull/6930/files#diff-8f184a7db1dec8fcc33fed7f6403327278bd8a4ab198f40a256a2a998e5fdccaR13

Reply:
Oops sorry I missed it!